datalake/src/main/java/com/sequenceiq/datalake/flow/dr/DatalakeDrSkipOptions.java

```diff
@@ -38,6 +46,7 @@ public boolean isSkipRangerMetadata() {
     @Override
     public String toString() {
         return "DatalakeDrSkipOptions{" +
+                "skipValidation=" + skipValidation +
                 "skipAtlasMetadata=" + skipAtlasMetadata +
```

Comment:
add comma in front of the `skipAtlasMetadata` for a better formatting

Reply:
Added. Thank you!!!